src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -87,9 +87,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
     Restore
     Main entry point for restoring packages
+    Returns a property with comma delimited projects whose imports
```

Comment:
is it comma delimited or semicolon delimited list?

Reply:
semicolon...nice catch :)